casatables/src/lib.rs

```diff
@@ -529,6 +529,50 @@ where
     )
 }
 
+// TableDesc
+
+pub struct TableDesc {
+    handle: *mut glue::GlueTableDesc,
+    exc_info: glue::ExcInfo,
+}
+
+impl TableDesc {
+    pub fn new(stype: &str) -> Self {
+        let ctype = glue::StringBridge::from_rust(stype);
+        let exc_info = unsafe { std::mem::zeroed::<glue::ExcInfo>() };
+
+        let handle = unsafe { glue::tabledesc_create(&ctype) };
+
+        // Ok(TableDesc {
+        //     handle,
+        //     exc_info
+        // })
```

Comment:
Once again this commented-out code should just be removed. I doubt that creation of a TableDesc will ever become fallible.